### ISSUE TITLE
Installing Application Inspector 1.4.18+ requires .Net 6, but it's not documented

### ISSUE THREAD

**Issue by liyuhang1997** [microsoft/ApplicationInspector#414]
**Describe the bug**
Installing Application Inspector 1.4.18+ requires .Net 6, but it's not documented.

https://github.com/microsoft/ApplicationInspector/blob/main/README.md
https://github.com/microsoft/ApplicationInspector/blob/main/JustRunIt.md

**To Reproduce**

```bash
C:\Users\isliy>dotnet tool install --global Microsoft.CST.ApplicationInspector.CLI --version 1.4.19
error NU1202: 包 Microsoft.CST.ApplicationInspector.CLI 1.4.19 与 net5.0 (.NETCoreApp,Version=v5.0) / any 不兼容。 包 Microsoft.CST.ApplicationInspector.CLI 1.4.19 支持: net6.0 (.NETCoreApp,Version=v6.0) / any
The tool package could not be restored.
Tool 'microsoft.cst.applicationinspector.cli' failed to install. This failure may have been caused by:

* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
* You mistyped the name of the tool.

For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool
```

**Comment by gfs** [comment]
Thanks for the report. Looks like I missed a couple places in the documentation.

**Comment by gfs** [comment]
Fixed in #415
